### DISCUSSION TITLE
关于alist连接onlyoffice令牌验证，如何关闭 JWT

### DISCUSSION THREAD

**Discussion by zhushen12580** [AlistGo/alist#3818] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/60245376/224528194-db725921-595e-46fb-a94e-25e6b8072486.png)

**Comment by Masterlovsky** [comment, marked as answer]
> ![image](https://user-images.githubusercontent.com/60245376/224528194-db725921-595e-46fb-a94e-25e6b8072486.png)
> 

emm，虽然我不知道这个问题的答案。但是如果实在不知道怎么搞可以下载7.11版本或者更早版本的onlyoffice，貌似是从7.12开始强制开启jwt的

**Comment by zhushen12580** [comment]
好的，谢谢。











只要你有梦想，全世界都为你让路。

---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年3月12日(周日) 下午4:32
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [alist-org/alist] 关于alist连接onlyoffice令牌验证，如何关闭 JWT (Discussion #3818)




  

  
emm，虽然我不知道这个问题的答案。但是如果实在不知道怎么搞可以下载7.11版本或者更早版本的onlyoffice，貌似是从7.12开始强制开启jwt的

**Reply by nxliubo** [reply]
你好，哥们儿，我正为alist连接不到本地部署的onlyoffice而困惑。能说说你是怎么做的吗？我是小白，不怕详细。谢谢！

**Reply by EarvinJones** [reply]
如果用的docker，加上这段`-e JWT_ENABLED=false`[https://helpcenter.onlyoffice.com/installation/docs-community-install-docker.aspx](url)，最近也在摸索，正好看到，不知道有没有帮助

**Reply by zhushen12580** [reply]
> 如果用的docker，加上这段`-e JWT_ENABLED=false`[https://helpcenter.onlyoffice.com/installation/docs-community-install-docker.aspx](url)，最近也在摸索，正好看到，不知道有没有帮助

谢谢，我用的Windows社区版，版本降到7.12以前，就没这个烦恼了。

**Reply by zhushen12580** [reply]
> 你好，哥们儿，我正为alist连接不到本地部署的onlyoffice而困惑。能说说你是怎么做的吗？我是小白，不怕详细。谢谢！

兄弟 翻翻讨论区有回答的，如果还没弄好，我可以提供帮助，不过有偿哦

**Reply by erduoya77** [reply]
我是 docker 跑的，加上以后变成了了“下载失败”，

**Comment by lemon-sundae** [comment]
echo onlyoffice-documentserver onlyoffice/jwt-enabled boolean false
这个命令可以关闭JWT
实测有效，新版本就能用

**Comment by moyan595** [comment]
# Uncomment strings below to enable the JSON Web Token validation.
      #- JWT_ENABLED=true
      - JWT_ENABLED=false
      #- JWT_SECRET=secret
      - JWT_SECRET=123456789A
      #- JWT_HEADER=Authorization
      - JWT_HEADER=Authorization
      #- JWT_IN_BODY=true
      - JWT_IN_BODY=true

**Reply by moyan595** [reply]
docker compose 配置文件里面修改下就行
